.github/workflows/integration_test.yaml

```diff
@@ -8,9 +8,6 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      chaos-app-label: app.kubernetes.io/name=indico
-      chaos-enabled: false
-      chaos-experiments: pod-delete
```

Comment:
@javierdelapuente  Could you please add a small note to the PR description stating that the chaos cleanup is also done (as this is unrelated to the original intent of the PR)?Thanks.